### ISSUE TITLE
:net Framework upgrade to 4.6.2

### ISSUE THREAD

**Issue by CSLAFan** [MarimerLLC/cslaforum#246]
Hi , 

I have various applications using customized version of CSLA 4.3.1 , .net 4.0 on VS 2012.
Is it possible to upgrade to .net 4.6.2 without upgrading CSLA. And Will I be able to use new features of 4.6.2 framework if use the CSLA 4.3.1.

**Comment by jonnybee** [comment]
Yes, you can compile Your solution for .NET 4.6 (ie change target framework) and you can use the features in .NET 4.6. 

Csla.Web.Mvc version 4.3.1 has a dependency to System.Web.Mvc version3.0.0 and I haven't tried to use this version with the latests MVC assemblies. Maybe someone else can chip in on this and how compatible the latest versions are with the builtin System.Web.Mvc in the .NET framework.

**Comment by CSLAFan** [comment]
Thank you Jonny.
